service-front/module/Application/src/Handler/AbstractHandler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Handler;
+
+use Mezzio\Template\TemplateRendererInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+abstract class AbstractHandler implements RequestHandlerInterface
```

Comment:
I don't think there's any benefit to having an AbstractHandler: every handler still needs to manage the TemplateRendererInterface dependency and it reduces the handlers' flexibility later.